app/html/menu.js

```diff
@@ -18,13 +18,28 @@ exports.create = function (api) {
     const hoverClass = Value('')
     const connectionClass = when(api.sbot.obs.connection, '', '-disconnected')
 
+    const sortByProp = (prop) => (a, b) => {
+      if (a[prop] < b[prop]) {
+        return -1
+      }
+      if (a[prop] > b[prop]) {
+        return 1
+      }
+      return 0
+    }
+
+    const menuItems = api.app.html.menuItem(api.app.sync.goTo)
+    const sortedMenuItems = computed([menuItems], items =>
```

Comment:
can you try this without `computed`, I don't thin menuItems is an observeable (as in it's not a value that's been set up to dynamically change)